src/diffusers/modeling_utils.py

```diff
@@ -23,6 +23,7 @@
 from huggingface_hub import hf_hub_download
 from huggingface_hub.utils import EntryNotFoundError, RepositoryNotFoundError, RevisionNotFoundError
 from requests import HTTPError
+from transformers.utils import is_accelerate_available
```

Comment:
We cannot do this either because `transformers` is not hard requirement :sweat_smile: 

I should have given you more details in my previous comments - very sorry that the feedback cycle takes so much time. Will try hard to reply faster here now. 

In short, can you copy this code https://github.com/huggingface/transformers/blob/e5b7cff5fe65eac9e54ba88fa3935b3270db0207/src/transformers/utils/import_utils.py#L528 into https://github.com/huggingface/diffusers/blob/main/src/diffusers/utils/import_utils.py

Reply:
Hey, thank you for clarifying that. I've just fixed it.